packages/graphql-server/src/plugins/__tests__/useRedwoodError.test.ts

```diff
@@ -12,9 +16,12 @@ jest.mock('../../makeMergedSchema', () => {
     ForbiddenError,
     RedwoodGraphQLError,
   } = require('@redwoodjs/graphql-server/dist/errors')
-  const { EmailValidationError, RedwoodError } = require('@redwoodjs/api')
 
   const { CurrencyResolver } = require('graphql-scalars')
+  const { RedwoodError, EmailValidationError } = require('@redwoodjs/api') as {
+    RedwoodError: typeof RedwoodErrorType
+    EmailValidationError: typeof EmailValidationErrorType
+  }
```

Comment:
This is pretty ugly. Highlights the need to switch over to vitest where this would be probably be a non-issue.